dev-client/src/screens/SoilScreen/TextureScreen.tsx

```diff
@@ -194,6 +194,11 @@ export const TextureScreen = (props: SoilPitInputScreenProps) => {
             onChange={onFragmentChange}
           />
         </Column>
+        <Fab
+          onPress={() => navigation.pop()}
+          leftIcon={<Icon name="check" />}
+          label={t('general.done')}
+        />
       </ScrollView>
```

Comment:
Good catch. This is fixed. Do you think it's worth having a <DoneButton> component? We use this 5 times.

Reply:
Eh, I feel ambivalent about it -- I think because each `DoneButton` would still do a different thing when pressed (right?), if we just have a standard Terraso-style Button component that would be fine, and all these Done buttons would just use that and the same "Done" string